bin/physlr-make

```diff
@@ -906,44 +910,44 @@ endif
 
 # Determine the maximum spanning tree.
 %.mst.tsv: %.tsv
-	$(python) $(bin)/physlr mst -V$V --prune-branches=10 --prune-bridges=10 $< >$@
+	$(python) $(bin)/physlr mst -V$V --prune-branches=$(prune_branches) --prune-bridges=$(prune_bridges) $< >$@
 
 # Separate a graph into its biconnected components by removing its cut vertices.
 %.bic.tsv: %.tsv
 	$(python) $(bin)/physlr biconnected-components -V$V $< >$@
 
 # Determine the bridge-removed graph from the overlap TSV.
 %.bridge-removed.tsv: %.tsv
-	$(time) $(python) $(bin)/physlr remove-bridges-graph --prune-branches=10 --prune-bridges=10 -s0 -V$V $< >$@
+	$(time) $(python) $(bin)/physlr remove-bridges-graph --prune-branches=$(prune_branches) --prune-bridges=$(prune_bridges) -s0 -V$V $< >$@
 
 # Report the junctions in a overlap/backbone graph TSV.
 %.junctions.tsv: %.tsv
-	$(time) $(python) $(bin)/physlr report-junctions-graph --prune-branches=10 --junction-depth=1 -s0 -V$V $< >$@
+	$(time) $(python) $(bin)/physlr report-junctions-graph --prune-branches=$(prune_branches) --junction-depth=1 -s0 -V$V $< >$@
 
 # Redo molecule separation on a list of junction-causing barcodes.
-%.mol2.tsv: %.tsv %.junctions.tsv
-	$(time) $(python) $(bin)/physlr molecules -V$V -t$t --separation-strategy=bc $^ >$@
+%.mol2.tsv: %.tsv
+	$(time) $(python) $(bin)/physlr molecules -V$V -t$t --separation-strategy=bc --round=2 $^ >$@
```

Comment:
These multi rounds will result in `barcode_mol` format or `barcode_mol_mol2`

Reply:
will be of format `bacode_mol1-mol2` if it's appropriate for all datasets

Reply:
That is gonna be a problem. Split minimizers uses regex of the form `barcode-num`

Reply:
I'll revert it back to `barcode_mol_mol2`. Is that fine?